[19707] Fix 'My Account' layout

floriank opened the pull request:
This uses a hack hack (an otherwise unnecessary `@extend`) to hack around the my
account layout, which breaks if certain plugins are used (myProject).

This relates to https://github.com/zurb/foundation-apps/issues/575, which
currently breaks layouts.

should also fix the issue documented in https://community.openproject.org/work_packages/19707.

I consider this a bandaid, not a satisfying solution, please treat it as such.

Commit:
patches the my account layout

This uses a hack hack (an otherwise uncessary @extend) to hack around the my
account layout.

This relates to https://github.com/zurb/foundation-apps/issues/575, which
currently breaks layouts.

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://floriank-fix-19707-my-account-layout.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-19707-my-account-layout.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

myabc commented:
@floriank can you instead try adding a `medium-up-2` class to the parent `grid-block`?

See http://foundation.zurb.com/apps/docs/#!/grid (Parent-level sizing)

floriank commented:
I'll to do that :+1:

Commit:
use an extension of `medium-up-2` to fix the issue

replaces the original solution with a suggestion from @myabc

[ci skip]

Commit:
remove the css hack

removes the css hack from before in order to add the class directly to DOM

[ci skip]

floriank commented:
Okay, I moved the hack into the DOM.

myabc commented:
@floriank I'll squish before merging, if that's ok.

floriank commented:
please do so.

myabc commented:
Squashed and merged as ba6df605752942884d848fef80a24b142fe94e2d.
